lib/block-supports/elements.php

```diff
@@ -5,6 +5,19 @@
  * @package gutenberg
  */
 
+/**
+ * Update the block content with elements class names.
+ *
+ * @deprecated 6.6.0 Use WP_Duotone_Gutenberg::register_duotone_support() instead.
+ *
+ * @param  string $block_content Rendered block content.
+ * @return string                Filtered block content.
+ */
+function gutenberg_render_elements_support( $block_content ) {
+	_deprecated_function( __FUNCTION__, '6.6.0', 'gutenberg_render_elements_support' );
```

Comment:
This refers to itself as the replacement in the third param.

Reply:
Thanks for catching that @peterwilsoncc! 🙈 

I've fixed it and the docblock in https://github.com/WordPress/gutenberg/pull/59538/commits/92ebee547c73bb5f30087787f6e39f379f1a6603